test/sshkit/client_key_api_test.exs

```diff
@@ -0,0 +1,82 @@
+    :asn1_NOVALUE
+  }  
+
+  @host_key {
+    :RSAPublicKey, 
+    21634204163197213132817109123906975906368888521544012567769262995559431966147970056259890368935740096079275379887017970430632559083119648736096672444000478892100121400122505155635695213610246722639150597148186404829574795017869184029845276838222700401896051725788665083080114314875103545837696279553436341967388240785773957395421170074137268759810304409727303757139265883118481355074238232002946450070460602471201997377623196017810991617729908588802783067540409316213603919494955068312762445623851275603318105356333614840694877780018045461415298887693169943421044958989561462337777142733008105081260079805978461159529, 
+    35
+  }
+  
+  setup do
+    support_dir = Path.join(File.cwd!, "test/support/keys")
+    key_file = Path.join(support_dir, "id_rsa")
+    known_hosts_tmpl = Path.join(support_dir, "known_hosts.tmpl")
+    known_hosts_file = Path.join(support_dir, "known_hosts")
+    File.cp!(known_hosts_tmpl, known_hosts_file)
+    on_exit(fn -> File.rm!(known_hosts_file) end)    
+    %{
+      known_hosts_file: known_hosts_file,
+      key_file: key_file
+    }
+  end
```

Comment:
I am always a bit cautious about file-system operations - esp. `rm`s - on the developer/local machine…

Also, always using the same paths prevents these test from being run in parallel. From the top of my head: Could we use tempfiles instead?